### PR TITLE
[Not support, don't merge. Can add some comments.]Adapt latest spec

### PR DESCRIPTION
**Pull request by kaixinjxq**
- Change sensor state attribute to activated.
- Split Accelerometer Sensor to three motion sensor.
- Modify copy right 2016 to 2017.

### CODE REVIEW

File: generic-sensor/sensors/GravitySensor_onerror-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<link rel="author" title="Xiuqi Jiang" href="mailto:xiuqix.jiang@intel.com">
+<link rel="help" href="https://w3c.github.io/accelerometer/#gravitysensor-interface">
+<script src="../resources/testharness.js"></script>
+<script src="../resources/testharnessreport.js"></script>
+<script src="./resources/generic-sensor-tests.js"></script>
+<div id="log"></div>
+<h2>Precondition</h2>
+<ol>
+  <li>
+    Disable the GravitySensor Sensor or run test on a device without GravitySensor Sensor.
```
Comment (Honry) [inline generic-sensor/sensors/GravitySensor_onerror-manual.https.html]: Please confirm if there's Gravity Sensor in underlying

File: generic-sensor/sensors/LinearAccelerationSensor_onerror-manual.https.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<link rel="author" title="Xiuqi Jiang" href="mailto:xiuqix.jiang@intel.com">
+<link rel="help" href="https://w3c.github.io/accelerometer/#linearaccelerationsensor-interface">
+<script src="../resources/testharness.js"></script>
+<script src="../resources/testharnessreport.js"></script>
+<script src="./resources/generic-sensor-tests.js"></script>
+<div id="log"></div>
+<h2>Precondition</h2>
+<ol>
+  <li>
+    Disable the LinearAccelerationSensor Sensor or run test on a device without LinearAccelerationSensor Sensor.
```
Comment (Honry) [inline generic-sensor/sensors/LinearAccelerationSensor_onerror-manual.https.html]: Ditto.

File: generic-sensor/sensors/resources/generic-sensor-tests.js [hunk 3]
```diff
@@ -81,14 +81,14 @@ function runGenericSensorTests(sensorType, verifyReading, readingToArray) {
   test(() => {
     let sensor = new sensorType();
     sensor.onerror = unreached;
-    assert_equals(sensor.state, "unconnected");
-  }, "default sensor.state is 'unconnected'");
+    assert_false(sensor.activated);
+  }, "default sensor.state is 'idle' not 'activated'");
```
Comment (Honry) [inline generic-sensor/sensors/resources/generic-sensor-tests.js]: sensor.state? still exist?

Comment (kaixinjxq) [inline generic-sensor/sensors/resources/generic-sensor-tests.js]: Define in spec like '[[state]] The current state of Sensor object which is one of "idle", "activating", or "activated". It is initially "idle".'

**Pull request closed without merging**